Sources/RInAppMessaging/RInAppMessaging.swift

```diff
@@ -18,6 +18,10 @@ import RSDKUtils
     internal static var swiftUIEventHandler: SwiftUIViewEventHandlerType? {
         dependencyManager?.resolve(type: SwiftUIViewEventHandlerType.self)
     }
+    /// Returns `true` when RMC module is integrated in the host app
+    internal static var isRMCEnvironment: Bool {
+        NSClassFromString("RMCInAppMessaging.RMCInAppMessaging") != nil
```

Comment:
`RMCInAppMessaging` is not a class.  Please update RMCSDK.

Reply:
done